src/main/java/duke/Command.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.task.Task;
+import duke.task.TaskManager;
+
+class Command {
+
+    private enum Commands {
```

Comment:
Perhaps a better approach would be to have enums to just store the `typeOfCommands` and then have some sort of a Parser Class that handles the parsing.